test/language_data/pubspec.lock

```diff
@@ -0,0 +1,320 @@
+# Generated by pub
```

Comment:
I appreciate a real generated file, but in the interest of making the test runtime shorter, could you cut down this file so the test doesn't start trying to pull up cves on so many things?  I see the test only looks for 

```
    DART_PRODUCTS = ["dio", "ffi", "path", "archive"]
```

So maybe just edit the fine to include those 4 and nothing else.  Or even just 1-2 of them if that's sufficient to test your parser's options.